router/core/src/main/scala/com/twitter/finagle/buoyant/FailureAccrualFactory.scala

```diff
@@ -0,0 +1,86 @@
+package com.twitter.finagle.buoyant
+
+import com.twitter.finagle.Stack.{Params, Role}
+import com.twitter.finagle.client.Transporter
+import com.twitter.finagle.liveness.{FailureAccrualPolicy, FailureAccrualFactory => FFailureAccrualFactory}
+import com.twitter.finagle.liveness.FailureAccrualFactory.Param
+import com.twitter.finagle.service.{ReqRep, ResponseClass, ResponseClassifier}
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.{ServiceFactory, Stack, Stackable, param}
+import com.twitter.logging.Level
+import com.twitter.util.Timer
+import io.buoyant.router.context.ResponseClassifierCtx
+
+object FailureAccrualFactory {
```

Comment:
it'd be nice to have a comment here like "A replacement for finagle's FailureAccrualFactory that ..."